cypress/e2e/content/content.spec.js

```diff
@@ -96,28 +96,22 @@ describe("Content Specs", () => {
     });
 
     it("Date Field", () => {
-      cy.get("#12-63ab04-0nkwcc input").click();
+      cy.get("#12-63ab04-0nkwcc button").click();
 
-      cy.get(
-        '.flatpickr-calendar:not(.hasTime) [aria-label="March 5, 2019"]'
-      ).click();
+      cy.get("[role=dialog]").find('[aria-label="Mar 5, 2019"]').click();
 
-      cy.get("#12-63ab04-0nkwcc input").should("have.value", "2019-03-05");
+      cy.get("#12-63ab04-0nkwcc input").should("have.value", "03/05/2019");
```

Comment:
Date format is REALLY important. we seem to be changing from a `-` delimiter to a `/` one. This could have significant implications to end consumption of these values. e.g. a customer has an API endpoint which relies on the date format. Can we specific the delimiter and keep it the same?

Reply:
@shrunyan Yea so this was my thought process.

On current prod we show the following
<img width="417" alt="Screen Shot 2022-07-06 at 4 40 49 PM" src="https://user-images.githubusercontent.com/10054410/177660662-1cf9d4eb-f74d-4d5f-a172-64ebd378bf43.png">

which then upon being sent to API gets resolved to:
date: "2019-03-05 00:00:00"
date_time: "2019-03-05 14:30:00.000000"

Given that our date time displays differently I figured there was no issue having what is displayed as long as it was being sent to the BE all the same.

All that being said I now realize it makes more sense for the value being sent directly to the BE to be displayed exactly how it is on the content editor so that users can see what date format to expect